src/main/java/ch/njol/skript/effects/EffConnect.java

```diff
@@ -29,65 +10,104 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.Utils;
 import ch.njol.util.Kleenean;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Connect")
-@Description("Connects a player to another bungeecord server")
-@Examples("connect all players to \"hub\"")
-@Since("2.3")
+@Description({
+	"Connect a player to a server running on your proxy, or any server supporting transfers. Read below for more information.",
+	"If the server is running Minecraft 1.20.5 or above, you may specify an IP and Port to transfer a player over to that server.",
+	"When transferring players using an IP, the transfer will not complete if the `accepts-transfers` option isn't enabled in `server.properties` for the server specified.",
+	"If the port is not provided, it will default to `25565`."
+})
+@Examples({
+	"connect all players to proxy server \"hub\"",
+	"transfer player to server \"my.server.com\"",
+	"transfer player to server \"localhost\" on port 25566"
+})
+@Since("2.3, INSERT VERSION (transfer)")
 public class EffConnect extends Effect {
 
 	public static final String BUNGEE_CHANNEL = "BungeeCord";
 	public static final String GET_SERVERS_CHANNEL = "GetServers";
 	public static final String CONNECT_CHANNEL = "Connect";
+	private static final boolean TRANSFER_METHOD_EXISTS = Skript.methodExists(Player.class, "transfer", String.class, int.class);
 
 	static {
 		Skript.registerEffect(EffConnect.class,
-				"connect %players% to [server] %string%",
-				"send %players% to server %string%"
+				"(transfer|send|connect) %players% to (proxy|bungeecord) server %string%",
+				"(transfer|send|connect) %players% to server %string% [on port %-number%]"
```

Comment:
Yes, this was discussed before. It is a breaking change.

Reply:
I missed the discussion, but wouldn't it be better to avoid said break?
Could potentially use "transfer" for transferring and "send|connect" for bungee